src/z_generate_repo.prog.abap

```diff
@@ -1008,6 +1019,26 @@ CLASS lcl_generator IMPLEMENTATION.
     print_logs( ).
   ENDMETHOD.
 
+  METHOD get_sval_data.
+    DATA: sval_data TYPE REF TO if_wb_object_data_model.
+    sval_data = NEW cl_wb_objtype_data( ).
+
+    NEW cl_wb_objtype_persist( )->if_wb_object_persist~get(
+      EXPORTING
+        p_object_key  = object_type
+        p_version     = 'A'
+        p_language    = 'E'
+      CHANGING
+        p_object_data = sval_data ).
+
+    sval_data->get_selected_data(
+      EXPORTING
+        p_data_selection = 'AL'
```

Comment:
I guess you want to avoid using the constant for `'AL'` to reduce depedencies, don't you?

Reply:
```suggestion
        p_data_selection = if_wb_object_data_selection_co=>c_all_data
```